chrome/content/zotero/zoteroPane.js

```diff
@@ -5682,6 +5683,11 @@ var ZoteroPane = new function()
 				continue;
 			}
 			
+			if (item.getField('title') === oldBaseName) {
```

Comment:
Can you add a comment explaining this?

I'm confused by this, because if I set a PDF title to "Foo" and then run Rename File from Parent Metadata, the file still gets named as "PDF". Is that not supposed to happen? If I change the test below to set the title to "Foo", it stays that way.

Reply:
Merged, but would still be good to clarify this and make sure it's doing what we want.

Reply:
Sorry, I may have been confusing "Rename File from Parent Metadata" and "Retrieve Metadata". I'm seeing what you're seeing for Rename File, and the behavior seems right.